docs/index.md

```diff
@@ -1,4 +1,43 @@
-# The Munich Quantum Toolkit
+```{only} html
+<p align="center">
+ <picture>
+   <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/cda-tum/mqt/main/docs/_static/mqt_light.png" width="80%">
+   <img src="https://raw.githubusercontent.com/cda-tum/mqt/main/docs/_static/mqt_dark.png" width="80%">
+ </picture>
+ </p>
+```
+
+# The MQT Handbook
+
+```{raw} latex
+\begin{abstract}
+```
+
+Quantum computers are becoming a reality.
```

Comment:
Not wrong, but in combination with the following sentence, this seems to suggest that applications are right around the corner. Maybe one could rephrase it to something like: "Universal large-scale quantum computers have been out of reach up to this point. However recent experimental advancements have shown that such quantum computers could become a reality."

It's not ideal as a first sentence, so one should probably put something before that. But I am drawing a blank as to what right now.

Reply:
Done in https://github.com/cda-tum/mqt/pull/10/commits/0f8a7402fe827d22aadaf7da4975c13205fe4a4c 👍🏼 
Tried to incorporate some version of your comment above. Please check again if that's better now :-)